scripts/regenerate_test_rustdocs.sh

```diff
@@ -27,7 +27,7 @@ for crate_pair in $(find "$TOPLEVEL/test_crates/" -maxdepth 1 -mindepth 1 -type
                 echo "Generating: $crate"
 
                 pushd "$TOPLEVEL/test_crates/$crate"
-                RUSTC_BOOTSTRAP=1 $RUSTDOC_CMD -- -Zunstable-options --document-private-items --document-hidden-items --output-format=json
+                RUSTC_BOOTSTRAP=1 $RUSTDOC_CMD -- -Zunstable-options --document-private-items --document-hidden-items --cap-lints allow --output-format=json
```

Comment:
Maybe use --cap-lints warn instead so the author still sees the warning?

I guess might not make sense for semver-checks since the fact it uses rustdoc is an implementation detail, though.

Reply:
Thanks, good idea.

By itself, that would get a bit annoying since I just pushed a crate that is going to emit a bunch of warnings every time this is executed.

So I'll add a bit more logic there and `allow` for that crate while using `warn` for everything else.